### PR TITLE
fix bug in SetSheetRow

### PR DESCRIPTION
**Pull request by leslie-corbalt**
This PR fixes a bug in main_test.go. This bug crashes main_test.go when number rows > 9.

Tested:
Added a test case, add_twelve_rows (for PR testing only), and main_test.go ran successfully.

### CODE REVIEW

File: main_test.go [hunk 1]
```diff
@@ -550,7 +550,7 @@ func TestRotate(t *testing.T) {
 				panic(err)
 			}
 			for idx, row := range tc.PasswordManagerIn {
-				err := f.SetSheetRow(sheetNamePasswordManager, "A"+string(rune('2'+idx)), &[]string{
+				err := f.SetSheetRow(sheetNamePasswordManager, "A"+fmt.Sprintf("%d", 2+idx), &[]string{
```
Comment (hundt-corbalt) [inline main_test.go:553]: nit: `fmt.Sprintf("A%d", 2+idx)` looks nicer to me, but leave it as-is if you prefer

File: main_test.go [hunk 2]
```diff
@@ -550,7 +550,7 @@ func TestRotate(t *testing.T) {
 				panic(err)
 			}
 			for idx, row := range tc.PasswordManagerIn {
-				err := f.SetSheetRow(sheetNamePasswordManager, "A"+string(rune('2'+idx)), &[]string{
```
Comment (hundt-corbalt) [inline main_test.go:553]: looks like the same bug is on line 535

**Review by hundt-corbalt — APPROVED** [review]
oops, good catch

File: main_test.go (hunk 1 above)
Comment (leslie-corbalt) [reply main_test.go:553]: Agree. Done.

File: main_test.go (hunk 2 above)
Comment (leslie-corbalt) [reply main_test.go:553]: Yes. Done.

**Review by hundt-corbalt — APPROVED** [review]
LGTM!

**Pull request merged**